cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricStore.java

```diff
@@ -158,6 +158,12 @@ public Cube get() {
       ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.APP,
                        Constants.Metrics.Tag.SPARK)));
 
+    // batch adapters - can only contain mapreduce for now
+    aggs.add(new DefaultAggregation(
+      ImmutableList.of(Constants.Metrics.Tag.NAMESPACE, Constants.Metrics.Tag.ADAPTER),
```

Comment:
@anwar6953  yeah, not even program-type, its aggregated and queried only at adapter level. @bdmogal can give more details.